app/models/todo.rb

```diff
@@ -54,10 +54,9 @@ def update_section_interaction_state
   end
 
   def todo_bot
-    if self.section_interaction.track.mentor.email.split("@")[0] == "bot"
-      sleep(1.seconds)
-      self.update(state: "completed") if self.state == "to_be_reviewed"
+    if self.section_interaction.track.mentor.email.split('@')[0] == 'bot'
```

Comment:
@RushilBasappa Try to avoid chaining more than 2 methods. Ideally, you would want to write something like `section_interaction.mentor.is_bot?`. To make this work you'll need to have a mentor method on `section_interaction` via `delegate` and user object will have a method `is_bot?`.